lib/commands/generate.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const fs = require('fs')
+const os = require('os')
+const {cwd, exists} = require('../util')
+const chalk = require('chalk')
+const path = require('path')
+const ignoreFiles = ['_navbar', '_coverpage', '_sidebar']
+
+// eslint-disable-next-line
+module.exports = function (path = '', sidebar) {
+  const cwdPath = cwd(path || '.')
+
+  if (exists(cwdPath)) {
+    if (sidebar) {
+      const sidebarPath = cwdPath + '/' + sidebar || '_sidebar.md'
+
+      if (!exists(sidebarPath)) {
+        genSidebar(cwdPath, sidebarPath)
+        console.log(chalk.green(`Generate sidebar file '${sidebar}' success.`))
+        return true
+      }
+
+      console.log(chalk.red(`${sidebarPath}`) + ' already exists.')
+      return false
```

Comment:
how about that let it rewrite `sidebar` when exists ? 
when we dont hv the permission to rewrite it, we throw warns.

Reply:
I have no intention of supporting rewriting. When it exists, if it needs to be regenerated, the user can just delete it themselves.

And the automatic generation is not always the same as the user's settings, still need to make your own changes

Reply:
I see.